src/components/BankTransactions/BankTransactionsBulkActions/BankTransactionsCategorizeAllButton.tsx

```diff
@@ -40,17 +41,16 @@ export const BankTransactionsCategorizeAllButton = () => {
           <VStack gap='xs'>
             <VStack gap='3xs'>
               <Label htmlFor={categorySelectId}>Select category</Label>
-              <CategorySelect
-                name={categorySelectId}
-                value={selectedCategory}
-                onChange={setSelectedCategory}
-                showTooltips={false}
-                excludeMatches={true}
+              <BankTransactionCategoryComboBox
+                inputId={categorySelectId}
+                selectedValue={selectedCategory}
+                onSelectedValueChange={setSelectedCategory}
+                includeSuggestedMatches={false}
               />
             </VStack>
-            {selectedCategory && (
+            {selectedCategory && isCategoryAsOption(selectedCategory) && (
               <Span>
-                {`This action will categorize ${count} selected transactions as ${selectedCategory?.payload?.display_name}.`}
+                {`This action will categorize ${count} selected transactions as ${selectedCategory?.original?.displayName}.`}
```

Comment:
Is it ever possible for `original` or `displayName` to be null, resulting to rendering a blank string?